iris-mpc-common/src/config/mod.rs

```diff
@@ -255,6 +255,9 @@ pub struct Config {
 
     #[serde(default = "default_batch_sync_polling_timeout_secs")]
     pub batch_sync_polling_timeout_secs: u64,
+
+    #[serde(default = "default_tokio_threads")]
```

Comment:
So the default here is to allow full overlap between Tokio's threads and the thread pool?

Reply:
To start yes. There might be phases with a lot of work in tokio: the remaining MPC computation besides dot products, high rate of small network packets, copying data around and book-keeping. To be seen what’s the optimal config and whether we need additional knobs (e.g. limits on the dot product pool).